DogeNet.BLL/Features/Account/CreateAccount/CreateAccountValidator.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="CreateAccountValidator.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Account.CreateAccount
+{
+    using FluentValidation;
+
+    public class CreateAccountValidator : AbstractValidator<CreateAccountModel>
+    {
+        private const int MaxLengthForPassword = 100;
```

Comment:
It would be better to move constants to separate file because you could access it in future commands

Reply:
done